conbench/templates/index.html

```diff
@@ -111,16 +117,7 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
       ],
       "language": {
           "searchPlaceholder": "commit hash works too",
-      },
-      "columns": [
-         { "width": "15%" },
-         { "width": "25%" },
-         { "width": "10%" },
-         { "width": "30%" },
-         { "width": "0%" },
-         { "width": "10%" },
-         { "width": "10%" },
-      ],
```

Comment:
This was hard to get "just right" for different screen sizes and I think the autoWidth is fine enough.